curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fad8e6148f310bba7890b1.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-38
 
 # --description--
 
-Center the `h1` and `p` elements by giving them a `margin` of `1em auto`. Then, center align the text as well.
+give the `h1` and `p` elements by giving them a `margin` of `1em auto`. Then, align their text in the `center` as well.
```

Comment:
```suggestion
Center the `h1` and `p` elements by giving them a `margin` of `1em auto`. Then, align their text in the `center` as well.
```
Not sure this clarifies things.

Reply:
True, it doesn't. 

I can change it to something like below, but it seems unnecessary to me.

> Give the `h1` and `p` a `margin` of `1em auto` to center them.